doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
  */
 ////
 
-calcurse(1)
+CALCURSE(1)
 ===========
 
-Name
+NAME
 ----
 
-calcurse - text-based organizer
+calcurse - terminal-based organizer for interactive and command line use
```

Comment:
Should we come up with a good concise description to use everywhere? Right now, the man page, documentation, the web site and distribution packages each use their own descriptions.

It should make the core features obvious, include the most relevant search keywords (calendar, scheduling, command line) and be as short as possible...

Reply:
I thought of this before and the only thing I do not like about it is that it's too long. I think we should aim for something with <80 characters but I cannot think of anything better right now.

Reply:
<80 is a tall order. This one is 77 characters long:

calcurse - customizable calendar app with reminders and command line features

Here 'interactive use'  is understood as opposed to  'command line'. I believe that 'app' is in common use for 'program' or 'application'.